### ISSUE TITLE
Remove root permissions requirement for embassy-sdk

### ISSUE THREAD

**Issue by ProofOfKeags** [Start9Labs/embassy-os#525]
SDK: Make it so that the package developer does not need to use `sudo` to run `embassy-sdk`. Developers should not need root permissions to build a package

Reported by @chrisguida

**Comment by k0gen** [comment]
I totally agree + 
```
embassy-sdk init
Filesystem I/O Error: /etc/embassy: Permission denied (os error 13)
```

**Comment by dr-bonez** [comment]
closed by #1501
